bats/tests/k8s/traefik.bats

```diff
@@ -1,9 +1,9 @@
 load '../helpers/load'
 
 local_setup() {
-    if using_networking_tunnel && ! using_windows_exe; then
+    if is_windows && ! using_windows_exe; then
```

Comment:
`using_windows_exe` is never true unless `is_windows`; see `defaults.bash` line 110.  Also the case for the next hunk.